src/Microsoft.Azure.WebJobs.Host/Queues/Listeners/HostMessageListenerFactory.cs

```diff
@@ -83,16 +80,9 @@ public Task<IListener> CreateAsync(CancellationToken cancellationToken)
         {
             ITriggerExecutor<IStorageQueueMessage> triggerExecutor = new HostMessageExecutor(_executor, _functionLookup, _functionInstanceLogger);
 
-            TimeSpan configuredMaximum = _queueConfiguration.MaxPollingInterval;
-            // Provide an upper bound on the maximum polling interval for run/abort from dashboard.
```

Comment:
Yeah, I see now the point of this maximum - without it when we set high MaxPollingInterval, Dashboard become less responsive. I think that complete removal of IDelayStrategy injection is a mistake. We can bring it back and create new strategy only if none was passed. Second option (and I think the best option) is to pass custom QueueProcessor in this case. Another option, that would be great too (for extensibility), is adding custom IDelayStrategy to QueueProcessor itself and fetch it in QueueListener constructor (but it's sort of completely separate feature).